osmaxx-symbology/QGIS/OSMaxx_M1.qgs

```diff
@@ -159,7 +159,7 @@
     <projections>1</projections>
     <destinationsrs>
       <spatialrefsys>
-        <proj4>+proj=merc +a=6378137 +b=6378137 +lat_ts=0.0 +lon_0=0.0 +x_0=0.0 +y_0=0 +k=1.0 +units=m +nadgrids=@null +wktext +no_defs</proj4>
+        <proj4>+proj=merc +a=6378137 +b=6378137 +lat_ts=0.0 +lon_0=0.0 +x_0=0.0 +y_0=0 +k=1.0 +units=m +nadgrids=@null +wktext  +no_defs</proj4>
```

Comment:
> I guess this extra space has been introduced by QGIS

Yeah, I suspect that QGIS on Linux is storing this differently from QGIS on Windows despite having the same version number being the same. Maybe due to different PROJ4 or GDAL/OGR behavior?

> Should we leave it in or removed it?

I'd leave it there, as saving the project again on Linux would re-introduce the space anyway. (I guess saving on Windows would remove it again. Meh.)